style.css

```diff
@@ -0,0 +1,474 @@
+  font-family: 'montserratsemibold';
+  position: relative;
+}
+
+.section__pic {
+  position: absolute;
+  float: left;
+  left: -10px;
+}
+
+.article {
+  position: relative;
+  float: left;
+  left: 1000px;
+  width: 31%;
+  color: white;
+  background-color: #e51937;
+
+}
+
+.article__info {
+	margin: 20px 30px 20px 20px;
+
+}
+
+.article__charset {
+	margin: 0;
+	display: flex;
+	flex-direction: column;
+}
+
+.article__top {
+	margin: 20px;
+	padding: 20px;
+}
+.article__title {
+	text-align: left;
+	padding: 10px;
+	font-size: 30px;
+}
+
+.article__text {
+	text-align: left;
+}
+
+
+.charset {
+	display: flex;
+	padding: 20px;
+	margin: 20px;
+}
+.charset__item {
+	margin: 0 10px;
+	width: 36px;
+  height: 53px;
+}
+
+.charset__item-one {
+	display: flex;
+	align-items: baseline;
+	text-align: left;
+}
+
+.charset__item-two {
+    display: flex;
+	flex-direction: column;
+}
+.charset__one-big {
+	font: 35px 'montserratsemibold';
+	text-align: left;
+	
+}
+
+.charset__one-small {
+	font: 20px 'montserratsemibold';
+}
+
+.charset__button-out {
+	border: none;
+	color: white;
+display: flex;
+}
+
+
+.charset__button-pic {
+	margin-right: 5px;
+}
+.charset__button-border {
+border: 2px solid white;
+	color: white;
+}
+
+.buttons {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	padding: 50px 20px 20px 20px;
+}
+
+.button__text {
+	text-transform: uppercase;
+}
+
+
+
+
+
+footer {
+	display: flex;
+	color: white;
+}
+
+.footer__box {
+	background-color: #E51937;
+	width: 440px;
+}
+
+.footer__inform {
+	padding: 20px;
+}
+
+.footer__car {
+
+margin:;
+}
+
+.footer__model {
+	z-index: 2;
+}
+
+.footer__footer {
+	flex-direction: column;
+	margin: 30px;
+}
+ 
+ .footer__title {
+ 	font: 30px 'montserratmedium';
+
+ }
+ .footer__text {
+	padding: 0 0 30px 0;
+}
+ 
+ 
+ .footer__info {
+ 
+	padding: 20px;
+ }
+ .footer__subtitle {
+ 	font: 30px 'montserratmedium';
+	padding: 20px;
+ }
+
+.footer__content {
+	font: 17px 'montserratbold';
+	
+}
+
+.footer__content-item {
+margin: 10px;
+}
+
+.content__items {
+margin: 0 40px;
+text-align: center;
+justify-content: center;
+display: flex;
+align-items: baseline;
+}
+
+.item {
+		align-items: center;
+	justify-content: center;
+	display: flex;
+}
+
+.footer__word-small {
+	font: 10px 'montserratregular';
+	opacity: 0.7;
+	
+}
+
+.footer__safe {
+	margin: 20px 0;
+	font: 50px 'montserratsemibold';
+}
+
+.safe {
+	align-items: center;
+	justify-content: center;
+	display: flex;
+}
+
+.safe__items {
+	margin: 40px 20px;
+}
+
+.safe__content-one {
+	align-items: center;
+	justify-content: center;
+	display: flex;
+}
+.safe__content-two {
+	font: 25px 'montserratmedium';
+}
+.footer__word {
+	margin-right: 10px;
+}
+
+.footer__small{
+	margin: 0;
+
+}
+.footer__button-out {
+	
+	border: none;
+}
+
+.footer__button-border {
+	color: white;
+}
+
+.footer__button-text {
+	text-transform: uppercase;
+	color: white;
+	font-size: 13px;
+}
+
+.footer__button-pic {
+	margin-right: 5px;
+}
+
+*/
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)